index.d.ts

```diff
@@ -2,6 +2,26 @@
 import File = require('vinyl');
 import * as globStream from 'glob-stream';
 import * as through from 'through2';
+import {EventEmitter} from 'events';
+
+export interface WatchOptions {
+    interval?: number;
+    debounceDelay?: number;
+    cwd?: string;
+    maxListeners?: Function;
```

Comment:
I didn't find this method in any docs. See https://github.com/gulpjs/glob-watcher/blob/195a14d51b734fd2ce762f44ec00de97ecb56da7/index.js and https://github.com/shama/gaze#properties.